### PR TITLE
Add additional postgres table, create association, update seeding script

### PR DESCRIPTION
**Pull request by christulin**
## Trello Ticket
https://trello.com/c/xTYlIP37/27-add-additional-postgres-table-and-create-associations

## Purpose of Pull Request
added a rooms table to differentiate from photos.  photos connected 1:M to rooms by `room_number`.  Updated seeding script to properly set up data

### CODE REVIEW

File: database/Models/index.js [hunk 1]
```diff
@@ -7,22 +7,19 @@ const sequelize = new Sequelize({
   database: 'sdc',
   password: 'test',
```
Comment (amosNash) [inline database/Models/index.js:8]: You suggested to me to put your password in your env file. I think you were going to do that at some point.

File: database/Models/index.js [hunk 2]
```diff
@@ -7,22 +7,19 @@ const sequelize = new Sequelize({
...
   port: 5432,
-  host: 'localhost'
+  host: 'localhost',
+  logging: false
 });
 
 const models = [
```
Comment (amosNash) [inline database/Models/index.js:14]: Could you run through lines 14-21? I'm a bit lost.

File: database/Models/index.js [hunk 3]
```diff
@@ -7,22 +7,19 @@ const sequelize = new Sequelize({
...
   require('./photos'),
-  //add other url model here
+  require('./rooms')
```
Comment (amosNash) [inline database/Models/index.js:16]: This is interesting how you are using require. Are you storing your files in this array models?

File: database/seed.js [hunk 4]
```diff
@@ -3,22 +3,45 @@ const { config } = require('../config.js');
...
+          const is_primary = false;
+          photos.push({ storage_url, name, caption, is_primary });
+        }
+      });
+    };
 
-  sequelize.models.photos.bulkCreate(objects);
+    try {
+      console.log(`finished ${run} run`);
+      await sequelize.models.rooms.bulkCreate(rooms);
```
Comment (amosNash) [inline database/seed.js:33]: Good work splitting your data into two tables.

File: database/seed.js [hunk 5]
```diff
@@ -3,22 +3,45 @@ const { config } = require('../config.js');
...
-      const caption = faker.commerce.productName();
-      const is_primary = i % 5 === 0;
-      objects.push({ storage_url, name, caption, is_primary });
-    });
-  };
+  for (let run = 0; run < 10000; run++) {
+    for (let i = 0; i < 5; i++) {
+      s3Photos.Contents.forEach((photo, j) => {
+        if (j % 5 === 0) {
+          const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
```
Comment (amosNash) [inline database/seed.js]: I think you could pull out the lines where you define storage_url, name, and caption from your conditional statement as you have that repeating.

File: database/seed.js [hunk 6]
```diff
@@ -13,18 +13,15 @@ const seed = async (database) => {
   for (let run = 0; run < 10000; run++) {
     for (let i = 0; i < 5; i++) {
       s3Photos.Contents.forEach((photo, j) => {
+        const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
```
Comment (amosNash) [inline database/seed.js:16]: You already did what I suggested!

**Review by amosNash — COMMENTED** [review]
Looks good!

File: database/Models/index.js (hunk 1 above)
Comment (christulin) [reply database/Models/index.js:8]: yes will get this done next PR!

File: database/Models/index.js (hunk 2 above)
Comment (christulin) [reply database/Models/index.js:14]: Pretty much each model file returns a function, which returns the instance of the sequelize model.  Each one is stored (Only two, but if project grew this seemed like a scalable way to structure it) in this `models` array.  and then below and looping over that array of functions, and invoking each function with the main sequelize object.  Happy to explain on stand up if unclear.. I didn't come up with this method, but thought it was neat.. can be found here: https://github.com/sequelize/express-example

File: database/Models/index.js (hunk 3 above)
Comment (christulin) [reply database/Models/index.js:16]: yeah the `require` statement returns what is exported from each model file, which is a function

**Pull request merged**